util/mini-tt2.pl

```diff
@@ -98,6 +98,16 @@
 my ($in_if, $in_else, $if_branch_hit);
 my ($in_block, $block, %blocks);
 my (%ctl_cmds, $prev_continuing_macro_line);
+my $modified = localtime((stat($infile))[9]);
+
+print $out <<"EOF";
```

Comment:
Better always use `_EOC_` for our heredoc terminator? Also, the double quotes can be omitted since it's the default.